docs/data/api/popover-positioner.json

```diff
@@ -52,6 +52,22 @@
   "imports": [
     "import { Popover } from '@base-ui-components/react/popover';\nconst PopoverPositioner = Popover.Positioner;"
   ],
+  "cssVariables": {
+    "--anchor-height": { "description": "The anchor's height.", "type": "number" },
+    "--anchor-width": { "description": "The anchor's width.", "type": "number" },
+    "--available-height": {
+      "description": "The available height of the positioner element.",
+      "type": "number"
+    },
+    "--available-width": {
+      "description": "The available width of the positioner element.",
+      "type": "number"
+    },
```

Comment:
These positioner vars need a tweak:
"The available height between the trigger and the edge of the viewport."
"The available width between the trigger and the edge of the viewport."